### PR TITLE
[New] Hackersploit Auto Security Updates Debian/Ubuntu & CentOS

### PR DESCRIPTION
**Pull request by hzoppetti**
Hackersploit autoupdates

### CODE REVIEW

File: docs/guides/security/basics/how-to-configure-automated-security-updates-ubuntu/index.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+        sudo apt update && sudo apt upgrade
+
+{{< note >}}
+This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you’re not familiar with the `sudo` command, see our [Users and Groups](/docs/guides/linux-users-and-groups) guide.
+{{< /note >}}
+
+## Install Unattended Upgrades
+
+You can set up automated security updates on Debian or Ubuntu by installing a helpful utility called `unattended-upgrades`.
```
Comment (andystevensname) [inline docs/guides/security/basics/how-to-configure-automated-security-updates-ubuntu/index.md:43]: Could remove Debian from this line

File: docs/guides/security/basics/how-to-configure-automated-security-updates-debian/index.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+        sudo apt update && sudo apt upgrade
+
+{{< note >}}
+This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you’re not familiar with the `sudo` command, see our [Users and Groups](/docs/guides/linux-users-and-groups) guide.
+{{< /note >}}
+
+## Install Unattended Upgrades
+
+You can set up automated security updates on Debian or Ubuntu by installing a helpful utility called `unattended-upgrades`.
```
Comment (andystevensname) [inline docs/guides/security/basics/how-to-configure-automated-security-updates-debian/index.md:43]: Can probably remove Ubuntu from this line

File: docs/guides/security/basics/how-to-configure-automated-security-updates-ubuntu/index.md (hunk 1 above)
Comment (hzoppetti) [reply docs/guides/security/basics/how-to-configure-automated-security-updates-ubuntu/index.md]: done

File: docs/guides/security/basics/how-to-configure-automated-security-updates-debian/index.md (hunk 2 above)
Comment (hzoppetti) [reply docs/guides/security/basics/how-to-configure-automated-security-updates-debian/index.md]: done

**Review by andystevensname — APPROVED** [review]

**Pull request merged**